README.md

```diff
@@ -80,8 +80,9 @@ rule "order new items urgent"
 when
     products_in_stock < 5,
 then
-    AND(order_more(10, true),
-    order_more(50))
+    order_more(10)
+then
```

Comment:
This was not intended to work :-( perhaps this is an error in parsing the rules.